modular_skyrat/modules/modular_weapons/code/company_and_or_faction_based/szot_dynamica/rifle.dm

```diff
@@ -0,0 +1,158 @@
+	suppressed_sound = 'modular_skyrat/modules/modular_weapons/sounds/suppressed_heavy.ogg'
+	can_suppress = FALSE
+
+	can_bayonet = FALSE
+
+	burst_size = 1
+	fire_delay = 0.6 SECONDS
+	actions_types = list()
+
+	recoil = 0.25
+	spread = 2.5
+	projectile_wound_bonus = -10
+	projectile_damage_multiplier = 0.85
+
+/obj/item/gun/ballistic/automatic/lanca/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/scope, range_modifier = 1.5)
+
+/obj/item/gun/ballistic/automatic/lanca/give_manufacturer_examine()
+	AddElement(/datum/element/manufacturer_examine, COMPANY_SZOT)
+
+/obj/item/gun/ballistic/automatic/lanca/examine_more(mob/user)
+	. = ..()
+
+	. += "The Lanca is a now relatively dated replacement for Kalashnikov pattern rifles \
+		adopted by states now combining to form the CIN. While the rifle that came before them \
+		had its benefits, leadership of many armies started to realize that the Kalashnikov-based \
+		rifles were really showing their age once the variants began reaching the thousands in serial. \
+		The solution was presented by a then new company, Szot Dynamica. This new rifle, not too \
+		unlike the one you are seeing now, adopted all of the latest technology of the time. Lightweight \
+		caseless ammunition, well known for its use in Sakhno rifles, as well as various electronics and \
+		other incredible technological advancements. These advancements may have already been around since \
+		before the creation of even the Sakhno, but the fact you're seeing this now fifty year old design \
+		must mean something, right?"
+
+	return .
+
+/obj/item/gun/ballistic/automatic/lanca/no_mag
+	spawnwithmagazine = FALSE
+
+// The AMR
+// This sounds a lot scarier than it actually is, you'll just have to trust me here
+
+/obj/item/gun/ballistic/automatic/wylom
+	name = "\improper Szot 'Wyłom' AMR"
+	desc = "A massive, outdated beast of an anti materiel rifle that was once in use by CIN military forces. Fires the devastating .60 Strela caseless round, the massively overperforming penetration of which being the reason this weapon was discontinued."
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/company_and_or_faction_based/szot_dynamica/guns_64.dmi'
+	base_pixel_x = -16 // This baby is 64 pixels wide
+	pixel_x = -16
+	righthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/szot_dynamica/inhands_64_left.dmi'
+	lefthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/szot_dynamica/inhands_64_right.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	worn_icon = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/szot_dynamica/guns_worn.dmi'
+	icon_state = "wylom"
+	inhand_icon_state = "wylom"
+	worn_icon_state = "wylom"
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = ITEM_SLOT_BACK
+
+	accepted_magazine_type = /obj/item/ammo_box/magazine/wylom
+	can_suppress = FALSE
+	can_bayonet = FALSE
```

Comment:
fix this post haste

Reply:
the gun does more damage just hitting people with it than most bayonets you can get would

Reply:
weak mindset